src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int side;
```

Comment:
I don't fully understand how we may describe right triangle with one side.
Seems we need 2 sides that create right corner